schema.graphql

```diff
@@ -15,13 +15,19 @@ type Token @entity {
   "Token address"
   address: Bytes!
   "First token trade block timestamp"
-  firstTradeTimestamp: BigInt!
+  firstTradeTimestamp: BigInt
   "Token name fetched by ERC20 contract call"
   name: String!
   "Token symbol fetched by contract call"
   symbol: String!
   "Token decimals fetched by contract call"
   decimals: Int!
+  "Derived price in ETH"
+  derivedETH: BigDecimal!
```

Comment:
nit: i guess this compes from Uni, but i would think if this is a price, it shold have the word price in it. `ethPrice` for example

Reply:
Makes sense. I kept that name from UniV3 but for our purpose we can renameit. I'll do.